loadtest/loadtest.go

```diff
@@ -245,6 +246,24 @@ func (lt *LoadTester) Status() *Status {
 	}
 }
 
+func (lt *LoadTester) InjectAction(action string) error {
+	lt.mut.Lock()
+	defer lt.mut.Unlock()
+
+	if lt.status.State != Running {
+		return ErrNotRunning
+	}
+
+	merr := merror.New()
+
+	for _, ctrl := range lt.activeControllers {
+		if resp := ctrl.InjectAction(action); resp.Err != nil {
+			merr.Append(resp.Err)
+		}
+	}
```

Comment:
Related to the previous comment, we could be keeping `lt.mut` locked for several seconds which isn't ideal.

Reply:
Agreed.  I now release the lock quickly by making a defensive copy of the active controllers slice.